kitchen/src/kitchen/core.clj

```diff
@@ -413,68 +407,83 @@
       (utils/exception->json-response e))))
 
 (defn websocket-handler
-  [request]
+  [{:keys [in out] :as request}]
   (swap! total-ws-requests inc)
-  (let [{:keys [in out] :as request}
-        (-> request
-            (update-in [:headers "x-cid"] (fn [cid] (or cid (str (UUID/randomUUID)))))
-            (assoc-in [:headers "x-kitchen-request-id"] (str (UUID/randomUUID))))]
-    (printlog request "Received websocket request:" request)
-    (swap! pending-ws-requests inc)
-    (async/go
-      (async/>! out "Connected to kitchen")
-      (loop []
-        (let [in-data (async/<! in)]
-          (printlog request "Received data on websocket:" in-data)
-          (if (or (str/blank? (str in-data)) (= "exit" in-data))
-            (do
-              (async/>! out "bye")
-              (printlog request "Closing connection.")
-              (async/close! out)
-              (swap! pending-ws-requests dec))
-            (do
-              (cond
-                (instance? ByteBuffer in-data)
-                (let [response-bytes (byte-array (.remaining in-data))]
-                  (.get in-data response-bytes)
-                  (async/>! out response-bytes))
-
-                (= "request-info" in-data)
-                (async/>! out (-> request request-info-handler :body))
-
-                (= "kitchen-state" in-data)
-                (async/>! out (-> request state-handler :body))
-
-                (and (string? in-data) (str/starts-with? in-data "chars-") (> (count in-data) (count "chars-")))
-                (let [num-chars-str (subs in-data (count "chars-"))
-                      num-chars-int (Integer/parseInt num-chars-str)
-                      chars (map char (range 65 91))
-                      string-data (->> (repeatedly #(rand-nth chars))
-                                       (take num-chars-int)
-                                       (reduce str))]
-                  (async/>! out string-data))
-
-                (and (string? in-data) (str/starts-with? in-data "bytes-") (> (count in-data) (count "bytes-")))
-                (let [num-bytes-str (subs in-data (count "bytes-"))
-                      num-bytes-int (Integer/parseInt num-bytes-str)
-                      byte-data (byte-array (take num-bytes-int (cycle (range 103))))]
-                  (async/>! out byte-data))
-
-                :else
-                (async/>! out in-data))
-              (recur))))))))
-
-(defn basic-auth-middleware [username password handler]
-  (fn [{:keys [uri] :as req}]
+  (printlog request "Received websocket request:" request)
```

Comment:
Done with lower-case.

The first `request` is used to extract the cid in `printlog`, the second to actually use in the log message.